### ISSUE TITLE
Fail to launch Spyder - updated the spyder output

### ISSUE THREAD

**Issue by sd2611** [spyder-ide/spyder#8211]
### Issue Report Checklist

* [ ] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [ ] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [ ] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [ ] Tried basic troubleshooting (if a bug/error)
    * [ ] Restarted Spyder
    * [ ] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [ ] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
I have installed Anaconda3-5.3.0-Windows-x86_64 on windows-7 x64 bit i3 processor laptop but whenever i tried to launch the Spyder from Anaconda, it failed to launch.

### What steps reproduce the problem?

1. 
2. 
3. 

### Paste Traceback/Error Below (if applicable)

![image](https://user-images.githubusercontent.com/44861647/48186155-324a6480-e35d-11e8-9f16-d770872b2c3b.png)

## Versions

* Spyder version: 3.3.1
* Python version: 3.7.0
* Qt version: 
* PyQt version: 
* Operating System name/version: Windows 7 x64 bit

### Dependencies

```
PASTE DEPENDENCIES HERE
```

**Comment by ccordoba12** [comment]
Please an Anaconda Prompt, run there

    spyder

and report back if you see any errors there.

**Comment by sd2611** [comment]
[image: image.png]
Please find above the output of spyder in anaconda prompt.

**Comment by ccordoba12** [comment]
You can't send the image from your email because Github won't accept it. You need to paste it directly here.

**Comment by sd2611** [comment]
[image: image.png]
Please find the ouput of spyder from anaconda prompt.

**Comment by ccordoba12** [comment]
As I said, you **can't** send images through your email.

**Comment by ccordoba12** [comment]
Closing because of lack of response.

**Comment by sd2611** [comment]
![spyder-output](https://user-images.githubusercontent.com/44861647/49018494-9a000c80-f1b1-11e8-8042-1570295bf198.PNG)
Sir,
Please look into the problem.

**Comment by sd2611** [comment]
Sir
I have updated the spyder output in getithub

**Comment by ccordoba12** [comment]
@CAM-Gerlach, could you help @sd2611 please? Thanks!

**Comment by CAM-Gerlach** [comment]
@sd2611 If you read the traceback, Spyder can't find any Qt binding package. Therefore, you should try installing one in your Spyder environment with ``conda install pyqt``. If that doesn't work, try ``conda install pyqt=5.9`` and ``conda install pyqt=5.6``. If that doesn't work, create a new environment with

```bash
conda create -n spyder-test-env python=3.6``
activate spyder-test-env
conda install spyder
spyder
```

If that doesn't work reinstall Anaconda. If *that* doesn't work, please try the steps listed in the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ), specifically the [Basic First Aid](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ#basic-first-aid-general-troubleshooting) and [Emergency CPR](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ#emergency-cpr-spyder-wont-launch) sections, which resolve the great majority of Spyder installation issues.

**Comment by ccordoba12** [comment]
Closing because of lack of response.

**Comment by sd2611** [comment]
I had tried all the options including several versions of Anaconda and
finally able to install the *Anaconda2-5.3.0-Windows-x86_64 *version in my
laptop.
Now the problem is that the version number of python is 2.7 which is an old
version. My fellow students are working on python version 3.7 which comes
along with *Anaconda3-5.3.0-Windows-x86_64. *

Please help me to resolve this problem.

**Comment by CAM-Gerlach** [comment]
If you can install one Anaconda version but not another, then the problem lies with Anaconda, not Spyder, so we don't really have direct control over it. As you say, given Python 2 has one year to live and its already on its deathbed in the PyData world (only took a decade...) you don't want to be using it if you can avoid it, and seeing you're a student its harder to learn and will teach you bad habits. 

What I recommend you do is as follows:

0. Uninstall any existing Anaconda version
1. Install ``Anaconda3-5.3.0-Windows-x86_64``. Try ``spyder``; if it fails, paste the _exact_ error message here (or refer to the above)
2. If it doesn't work, run the following commands to create a fresh environment for Spyder:
 
  ```bash
   conda create -n spyder-env python=3.6
   activate spyder-env
   conda install spyder
   spyder
   ```

3. If that still won't work, paste the error message here (if different) and try a different Qt binding version: ``conda install pyqt=5.6``
4. Paste the output of ``conda list qt`` here

You can also try the steps listed in the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ), specifically the [Basic First Aid](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ#basic-first-aid-general-troubleshooting) and [Emergency CPR](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ#emergency-cpr-spyder-wont-launch) sections, which resolve the great majority of Spyder installation issues.

**Comment by sd2611** [comment]
Let me try again above 0-4 steps one by one and then update you the result.
Please bear with me.

**Comment by sd2611** [comment]
Sir,
As per your guide, I have executed the below points and also included the output for your reference and verification.
 
0. Uninstall any existing Anaconda version
----Uninstalled the previous version. Done

1.  Install Anaconda3-5.3.0-Windows-x86_64. Try spyder; if it fails, paste the exact error message here (or refer to the above)
----Installed this version. But when I have clicked on this option, nothing is activated.
![image](https://user-images.githubusercontent.com/44861647/50299070-ae7caf80-04a6-11e9-9636-3dc9b358e926.png)

2. If it doesn't work, run the following commands to create a fresh environment for Spyder:
 conda create -n spyder-env python=3.6
 activate spyder-env
 conda install spyder
 spyder

![image](https://user-images.githubusercontent.com/44861647/50299185-fac7ef80-04a6-11e9-824a-9e6e698d9934.png)

![image](https://user-images.githubusercontent.com/44861647/50299218-13380a00-04a7-11e9-9b33-7d0da8d28dc9.png)

![image](https://user-images.githubusercontent.com/44861647/50299239-234fe980-04a7-11e9-8e82-4a312be19ada.png)

3.  If that still won't work, paste the error message here (if different) and try a different Qt binding version: conda install pyqt=5.6

![image](https://user-images.githubusercontent.com/44861647/50299270-32369c00-04a7-11e9-8206-07b40b49a4db.png)

4. Paste the output of conda list qt here
![image](https://user-images.githubusercontent.com/44861647/50299293-411d4e80-04a7-11e9-9ca0-1e2a19cdba1f.png)

**Comment by sd2611** [comment]
Sir,

I have updated the output of 0-4 points as mentioned above.
Awaiting your valuable suggestion to resolve this deadlock.

**Comment by CAM-Gerlach** [comment]
@sd2611 Okay, thanks very much for providing all that output above. The key error message is the ``broker pipe ended`` one, which which is not uncommon on Windows, especially Windows 10 and is due to an issue with your operating system, security software/anitvirus/firewall, or your graphics drivers (see e.g. #7690 for more details). It can't be fixed within Spyder because it is typically due to an interaction between your OS/drivers/antimalware and our 3rd party GUI library Qt, but we've found the following steps likely to fix it on your end:

0. Did you try launching Spyder after downgrading PyQt? Did you get the same ``broker pipe ended`` error? I don't see anything after your last screenshot of downgrading PyQt so I want to confirm. That was reported to be the most common fix that worked; I had the very same problem on my own Windows 10 machine and that's what fixed it.

1. Try starting with the the different opengl implementations; either ``spyder --opengl software`` and ``spyder --opengl desktop``. If you find one that works, you can set it permanently in the Spyder Preferences, under ``General > Advanced Settings``.

2. Try temporarily disabling any anti-malware, antivirus, firewall, system protection, Windows Defender, or other security software you have installed, and see if launching Spyder works.

3. Update your operating system and particularly your graphics drivers, as driver incompatibilities is a common cause

4. If your system has multiple GPUs (e.g. Intel integrated and Nvidia discrete), try disabling each one and see if it works on the other.

5. Try running the Anaconda command prompt as administrator, running ``activate spyder-env`` and trying Spyder again. Also, try installing Anaconda itself as administrator (i.e. "for all users") if you can. Hopefully it won't come to this.

Let me know your results and if you see the "broker pipe" error or another one with each of these steps. Thanks, and best of luck!

**Comment by sd2611** [comment]
Let me try out these steps and revert back to you.
Thank you so much.
